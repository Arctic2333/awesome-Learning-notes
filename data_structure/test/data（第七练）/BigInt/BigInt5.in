9237194723974912739472397498213794236487231648621834682137648263187468271364872136846238648213648623846823648236486213894612032485032840583240850923840598324098502398453240580349850932480598230458032485093428509348058304985093248508324058032485034805834029580348503480583048503485038405803498509348252038503845038405830428509342808508923746892368417684687468237648723746820294387598347597349579345892379045897349825702348524
23490850934280503248502348095292371947239749127394723974982137942364872316486218346821376482631874682713648721368462386482136486238468236482364862138946120324850328405832408509238405983240985023984532405803498509324805982304580324850934285093480583049850932485083240580324850348058340295803485034805830485034850384058034985093482520385038450384058304285093428085089237468923684176846874682376487237468202943875983475973495793458923790458973492342356779523490859032485927398932847876
