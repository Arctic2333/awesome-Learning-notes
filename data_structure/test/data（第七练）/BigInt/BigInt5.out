23490850934280503248502348095292371947239749127394723974991375137088847229225690744319590276868361914362270556050600034745323954509833108619211100787159768948697152054068894723133018015726017864567773256727339107648904484703033565431284136025961181280308964970176669089672908653043433544311809092838315519840684413638383488574065568888523488789861802794441680123593082507329514605356217490885410984360571361560670944211144517205744084846571839939706358869383238078383277224635196400
